device-types/Juniper/MX960.yaml

```diff
@@ -1,7 +1,32 @@
 ---
 manufacturer: Juniper
-model: MX960
+model: MX960xx
```

Comment:
This doesn't look correct.

What is the xx for?  What does the inventory show on the actual CLI for a model number?